django_ipam/base/models.py

```diff
@@ -33,6 +34,14 @@ def clean(self):
             if self.id != subnet["id"] and ip_network(self.subnet).overlaps(subnet["subnet"]):
                 raise ValidationError({'subnet': _('Subnet overlaps with %s') % (subnet["subnet"])})
 
+    def ip_request(self):
```

Comment:
why don't we rename this `request_ip` instead? Doesn't it make much more sense to call it so? We are requesting a new ip after all :-)